### PR TITLE
[Portuguese] Translate guide/README.md

### PR DESCRIPTION
**Pull request by kr4t0z**

### CODE REVIEW

File: site/docs/pt/guide/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+1. Documentação do grammY de alto nível _(this website)_
+2. [Referência da API de baixo nível do grammY](https://deno.land/x/grammy/mod.ts)
+3. [Referência da API HTTP bruta do Telegram](https://core.telegram.org/bots/api)
+
+**A primeira parte** (você está olhando pra ela!) explica como os bots funcionam e como usar o grammY.
+É o que você vai ver com mais frequência.
+A seção _Aprenda_ é sempre um bom start.
+Mas também passe pela nossa grande coleção de _Plugins_, e dê uma olhada nos _Exemplos_.
+
+**A segunda parte** é a  [referência da API do grammY](https://deno.land/x/grammy/mod.ts), linkado no topo da página.
```
Comment (kr4t0z) [inline site/docs/pt/guide/README.md:30]:
```suggestion
**A segunda parte** é a [referência da API do grammY](https://deno.land/x/grammy/mod.ts), linkado no topo da página.
```

File: site/docs/pt/guide/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Dê uma olhada na [Introdução](./introduction) se você é novo no mundo dos bots do Telegram, especialmente se você tem pouca experiência com programação.
+
+Se você sabe como escrever uma aplicação em Node.js ou Deno ou já criou algum bot do Telegram com outro framework para bots, comece em poucos minutos com o nosso guia de [Primeiros Passos](./getting-started).
+
+## Sobre a documentação
+
+A documentação do grammY é dividida em três camadas.
+
+1. Documentação do grammY de alto nível _(this website)_
```
Comment (roziscoding) [inline site/docs/pt/guide/README.md:21]:
```suggestion
1. Documentação do grammY de alto nível _(este site)_
```

File: site/docs/pt/guide/README.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+2. [Referência da API de baixo nível do grammY](https://deno.land/x/grammy/mod.ts)
+3. [Referência da API HTTP bruta do Telegram](https://core.telegram.org/bots/api)
+
+**A primeira parte** (você está olhando pra ela!) explica como os bots funcionam e como usar o grammY.
+É o que você vai ver com mais frequência.
+A seção _Aprenda_ é sempre um bom start.
+Mas também passe pela nossa grande coleção de _Plugins_, e dê uma olhada nos _Exemplos_.
```
Comment (roziscoding) [inline site/docs/pt/guide/README.md:28]:
```suggestion
A seção _Aprenda_ é sempre um bom start, mas também vale a pena conferir nossa grande coleção de _Plugins_, e dar uma olhada nos _Exemplos_.
```

File: site/docs/pt/guide/README.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+**A segunda parte** é a [referência da API do grammY](https://deno.land/x/grammy/mod.ts), linkado no topo da página.
+É uma visão geral bem detalhada de cada pedacinho de código que o grammY fornece.
```
Comment (roziscoding) [inline site/docs/pt/guide/README.md:31]:
```suggestion

**A segunda parte** é a [referência de API do grammY](https://deno.land/x/grammy/mod.ts), linkado no topo da página.
É uma visão detalhada de cada pedacinho de código que o grammY fornece.
```

Although "detailed overview" works in English, "visão geral" is basically the opposite of "detalhado", so it doesn't really work the same in Portuguese

File: site/docs/pt/guide/README.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+É gerada automaticamente a partir do código do grammY e contém todas as explicações úteis das dicas de ferramenta, que são geralmente encontradas quando você passa seu cursor sobre um elemento do grammY em um editor de código.
```
Comment (roziscoding) [inline site/docs/pt/guide/README.md:32]:
```suggestion
É gerada automaticamente a partir do código do grammY e contém todas as explicações úteis de dicas de ferramenta, que são geralmente encontradas quando você passa seu cursor sobre um elemento do grammY em um editor de código.
```

Idk, this still feels weird. Tbf I've never seen tooltip translated like this 🤔

File: site/docs/pt/guide/README.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+**A terceira parte** é fornecida pelo Telegram e lista as definições brutas da [API HTTP](https://core.telegram.org/bots/api) que o grammY vai se conectar debaixo dos panos.
```
Comment (roziscoding) [inline site/docs/pt/guide/README.md:34]:
```suggestion
**A terceira parte** é fornecida pelo Telegram e lista as definições brutas da [API HTTP](https://core.telegram.org/bots/api) à qual o grammY vai se conectar por debaixo dos panos.
```

File: site/docs/pt/guide/README.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+A referência da API do grammY aponta pra lá sempre que faz sentido.
```
Comment (roziscoding) [inline site/docs/pt/guide/README.md:35]:
```suggestion
A referência de API do grammY aponta pra lá sempre que faz sentido.
```

File: site/docs/pt/guide/README.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+Dê uma olhada lá quando estiver interessado em opções detalhadas que você pode passar para chamadas de API.
+
+::: dica: Entre na Comunidade!
```
Comment (roziscoding) [inline site/docs/pt/guide/README.md:38]:
```suggestion
::: tip: Entre na Comunidade!
```

We should not translate these prefixes, as they are actually special markdown code that generates the colored portions on the website

File: site/docs/pt/guide/README.md [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+Nós temos um amigável [chat da comunidade](https://t.me/grammyjs) no Telegram que é bem receptivo com novos membros. (Você pode achar o chat russo [aqui](https://t.me/grammyjs_ru).)
```
Comment (roziscoding) [inline site/docs/pt/guide/README.md:39]:
```suggestion
Nós temos um [chat da comunidade](https://t.me/grammyjs) amigável no Telegram que é bem receptivo com novos membros. (Você pode encontrar o chat russo [aqui](https://t.me/grammyjs_ru).)
```

File: site/docs/pt/guide/README.md [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+Junte-se a nós para receber assistência, fazer perguntas e aprender dicas e macetes para o seu próximo projeto com bot.
```
Comment (roziscoding) [inline site/docs/pt/guide/README.md:40]:
```suggestion
Junte-se a nós para receber ajuda, fazer perguntas e aprender dicas e macetes para o seu próximo projeto de bot.
```

**Review by roziscoding — CHANGES_REQUESTED** [review]

File: site/docs/pt/guide/README.md (hunk 5 above)
Comment (kr4t0z) [reply site/docs/pt/guide/README.md:31]: yeah, I've spent a few minutes trying to find the best fit for it. Still weird. We could flag that to think about later.

File: site/docs/pt/guide/README.md [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+
+**A segunda parte** é a [referência de API do grammY](https://deno.land/x/grammy/mod.ts), linkado no topo da página.
+É uma visão detalhada de cada pedacinho de código que o grammY fornece.
+É gerada automaticamente a partir do código do grammY e contém todas as explicações úteis das dicas de ferramenta, que são geralmente encontradas quando você passa seu cursor sobre um elemento do grammY em um editor de código.
+
+**A terceira parte** é fornecida pelo Telegram e lista as definições brutas da [API HTTP](https://core.telegram.org/bots/api) à qual o grammY vai se conectar por debaixo dos panos.
+A referência de API do grammY aponta pra lá sempre que faz sentido.
+Dê uma olhada lá quando estiver interessado em opções detalhadas que você pode passar para chamadas de API.
+
+::: tip: Entre na Comunidade!
```
Comment (kr4t0z) [inline site/docs/pt/guide/README.md:37]:
```suggestion
::: tip Entre na Comunidade!
```

File: site/docs/pt/guide/README.md (hunk 5 above)
Comment (kr4t0z) [reply site/docs/pt/guide/README.md:31]:
```suggestion
É gerada automaticamente a partir do código do grammY e contém todas as explicações úteis de descrições de ferramenta, que são geralmente encontradas quando você passa seu cursor sobre um elemento do grammY em um editor de código.
```

maybe like that? Or rephrase the entire sentence to find a better arrangement of words with the same meaning?

File: site/docs/pt/guide/README.md [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+## O Que É o grammY?
+
+O grammY é um framework para criar bots do telegram.
+Ele pode ser usado com TypeScript e JavaScript e roda em Node.js, Deno, e no navegador.
+
+Dê uma olhada na [Introdução](./introduction) se você está começando no mundo dos bots do Telegram, especialmente se você tem pouca experiência com programação.
+
+Se você sabe como escrever uma aplicação em Node.js ou Deno ou já criou algum bot do Telegram com outro framework para bots, comece em poucos minutos com o nosso guia de [Primeiros Passos](./getting-started).
+
+## Sobre a documentação
```
Comment (roziscoding) [inline site/docs/pt/guide/README.md:17]:
```suggestion
## Sobre a Documentação
```

File: site/docs/pt/guide/README.md [hunk 13]
```diff
@@ -0,0 +1,40 @@
...
+2. [Referência da API de baixo nível do grammY](https://deno.land/x/grammy/mod.ts)
+3. [Referência da API HTTP bruta do Telegram](https://core.telegram.org/bots/api)
+
+**A primeira parte** (você está olhando pra ela!) explica como os bots funcionam e como usar o grammY.
+É o que você vai ver com mais frequência.
+A seção _Aprenda_ é sempre um bom start, mas também vale a pena conferir nossa grande coleção de _Plugins_, e dar uma olhada nos _Exemplos_.
+
+**A segunda parte** é a [referência de API do grammY](https://deno.land/x/grammy/mod.ts), linkado no topo da página.
+É uma visão detalhada de cada pedacinho de código que o grammY fornece.
+É gerada automaticamente a partir do código do grammY e contém todas as explicações úteis das dicas de ferramenta, que são geralmente encontradas quando você passa seu cursor sobre um elemento do grammY em um editor de código.
```
Comment (roziscoding) [inline site/docs/pt/guide/README.md:31]:
```suggestion
É gerada automaticamente a partir do código do grammY e contém todas as explicações úteis que são geralmente encontradas quando você passa seu cursor sobre um elemento do grammY em um editor de código.
```

What about this? Remove the word entirely?

**Review by roziscoding — APPROVED** [review]
LGTM as soon as we decide about how to deal with "tooltip"

File: site/docs/pt/guide/README.md (hunk 13 above)
Comment (kr4t0z) [reply site/docs/pt/guide/README.md:31]: perfect!

**Pull request merged**